packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -152,18 +176,48 @@ export default decorate([
       </FormGrid.Row>
       {value.nbNodes > 0 &&
         value.staticIpAddress && [
-          <FormGrid.Row key='controlPlaneIpAddrRow'>
-            <label>{_('recipeControlPlaneIpAddress')}</label>
-            <input
-              className='form-control'
-              name='controlPlaneIpAddress'
-              onChange={effects.onChangeValue}
-              placeholder={formatMessage(messages.recipeControlPlaneIpAddress)}
-              required
-              type='text'
-              value={value.controlPlaneIpAddress}
-            />
-          </FormGrid.Row>,
+          value.highAvailability && [
+            [...Array(+3)].map((v, i) => (
```

Comment:
Instead of using spread operator, this syntax should also work.
```suggestion
            Array.from({length: 3}).map((v, i) => (
```